packages/docs/static/data/datatypeDefinitions/meta.json

```diff
@@ -218,10 +218,28 @@
       "min": 0,
       "max": "1",
       "short": "Optional account reference that can be used for sub-project compartments.",
-      "definition": "@deprecated Use Meta.compartment instead",
+      "definition": "@deprecated Use Meta.accounts instead",
```

Comment:
Changes in this file appear to be missing changes from the `compartment`/`account`/`accounts` cleanup.